### ISSUE TITLE
Pig sounds persist after April Fools' Day

### ISSUE THREAD

**Issue by Beetrootmonkey** [DarkPacks/SkyFactory-4#1875]
## Bug Description
The player model displayed a pig snout on April Fool's Day. The game also frequently played a Pig sound.
It's now 2021-04-03, and while the snout is gone, the sounds still persist.

If this is intended, please provide instructions on how to disable this. While this might have been funny for one day, I don't enjoy this feature enough to want it enabled forever. Yes, I could just disable sounds alltogether, but I really just want this specific sound/feature gone.

## Steps to Reproduce (for bugs)
1. Open up Single Player or join any MP server.
2. Wait for up to 10 seconds.
3. Although there are propably no Pigs around, the game will still play the typical Pig sound. 

## World Information
* World Preset: Default
* Prestige Enabled: No
* Modpack Version world created in: 4.2.2
* Additional Content Installed: No

## Client Information
* Modpack Version: 4.2.2
* Java Version: 1.8.0.261
* Launcher Used: CurseForge
* Memory Allocated: 6 GB
* Server/LAN/Single Player: MP
* Optifine Installed: No
* Shaders Enabled: No

**Comment by sam-kirby** [comment]
This effect is added by iChun/iChunUtil and cannot be disabled. If we could, we would as we have with all other configurable holiday events.

The snout effect is handled by the client, whilst the sound is played by the server.

iChun only checks the date once at startup, so if you are still hearing the sound effect it likely means your server restarted on the 1st; restarting again should fix it.

**Comment by Beetrootmonkey** [comment]
Thanks for your reply, even though the bot had already closed this issue.

See, I couldn't even find out what mod brought this effect. I eventually figured out myself that a server restart might fix the issue, since maybe the mod only checked the date on start-up (that's what I'd do, anyway).

You're right, we started playing this modpack on 1st of april, meaning we would have kept hearing this sound until the server eventually crashed or we stopped playing the pack.
